ddtrace/contrib/asgi/middleware.py

```diff
@@ -28,7 +29,12 @@
 
 config._add(
     "asgi",
-    dict(service_name=config._get_service(default="asgi"), request_span_name="asgi.request", distributed_tracing=True),
+    dict(
+        service_name=config._get_service(default="asgi"),
+        request_span_name="asgi.request",
+        distributed_tracing=True,
+        trace_asgi_websocket=os.getenv("DD_ASGI_TRACE_WEBSOCKET", default=False),
```

Comment:
We should rename this configuration to better align with existing conventions. 

We should also make `trace_asgi_websocket` "internal". This will give use flexibility and allow us to refactor this code. Setting this config via envar should be sufficient.

```suggestion
        _trace_asgi_websocket=os.getenv("DD_TRACE_ASGI_WEBSOCKET", default=False),
```